charts/opensearch-dashboards/values.yaml

```diff
@@ -155,3 +155,16 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+lifecycle: {}
+  # preStop:
+    # exec:
+  #     command: ["/bin/sh", "-c", "echo Hello from the postStart handler > /usr/share/message"]
```

Comment:
Can we have something more formalized here? 

I'm spitballing but "Pod terminating. Beginning pod lifecycle commands.". Should we spit this out to `stderr`?

Reply:
@DandyDeveloper I just copy and pasted what is in the opensearch helm chart here any change should probably be done in both charts, and should be agreed upon.

Reply:
@DandyDeveloper was the config section int the opensearch chart autogenerated? or hand rolled? https://github.com/opensearch-project/helm-charts/tree/main/charts/opensearch#configuration I notice that opensearch-dashboards is missing an equivalent.

Reply:
@bluehawk27 This was hand rolled out. The config section in dashboards is pending. We have an issue open for this #129